### DISCUSSION TITLE
Please add important warning in guide for Optane, Raid, Soft raid users

### DISCUSSION THREAD

**Discussion by dimka4996** [xCuri0/ReBarUEFI#170] [General]
If you using intel optane, raid on intel controllers, please turn them in intel storage management software and set ahci mode in bios before modding bios or all data will be lost.

**Comment by xCuri0** [comment]
What's the reason for this ? Shouldn't it be no different than installing a regular BIOS update ?

**Comment by dimka4996** [comment]
> What's the reason for this ? Shouldn't it be no different than installing a regular BIOS update ?

After messing with all bios flash with programmer and setting bar size sata controller can be appear like new to the system and OS will install driver for him again and broke all metadata on storage. When metadata broken volumes will be shown like raw unformatted. So its just warning, disable all these raids and optane to prevent data lost

**Comment by xCuri0** [comment]
Wouldn't a BIOS update from OEM do the same though ?
